### ISSUE TITLE
oda restore issue

### ISSUE THREAD

**Issue by dikim33** [oscar-cluster/oscar#302]
**Reported by bli on 21 Mar 2007 08:16 UTC**
Tried to restore a dump:

```
# ./scripts/oda restore /home/root/backup/oscar.2007-03-20.18911
Illegal division by zero at ./scripts/oda line 360.
```

When using the oda CLI, the `restore` command does not take full path to the dump file, so I had to copy the dump file to `tmp/oda_dump` then it can restore the DB.

This message is a bit ambigious:

```
oda> restore
List of dump files: oscar.2007-03-20.18911
Enter the name of the dump file to restore: oscar.2007-03-20.18911
The database(oscar) already exist!!!
For the clean restore, please remove the database(oscar) or recreate it

Do you want to proceed (y/n)? y
```

If I choose to proceed (y) does that mean it will clear the old database?

**Comment by dikim33** [comment]
**Comment by dikim on 22 Mar 2007 18:26 UTC**
(In [5754]) scripts/oda restore can take a dumpfile with its full path now.
Bascially a user does not have to input a full path of the dump file if
the dump file(s) is saved in the default place
($OSCAR_HOME/tmp/oda_dump). More conveniently, if "oda restore" is
called without any dump file, it will list up users' dump files in the
default place. Even when there is no dump file in the default place, it
will create a dump file if users want.
I have put the more verbose comments in the command line output to
clarify whant happens when there is already an existing OSCAR database.
I hope this fixes the ticket #302 and please see #302 for more
information.

**Comment by dikim33** [comment]
**Comment by dikim on 22 Mar 2007 23:42 UTC**
(In [5755]) Clarify the "oda restore" issue and add a function to remove oda "oda
remove_oda". I hope this fixes #302
